### ISSUE TITLE
Block spacing control no longer displays "link" button

### ISSUE THREAD

**Issue by webmandesign** [WordPress/gutenberg#53151]
### Description

Please re-enable linking button for block spacing control. And make it enabled by default, if possible.

In my opinion setting up different horizontal and vertical block spacing/gap is more of an "advanced" approach, and in most cases setting up a single value for both is sufficient. (While I still want to keep the option to set separate gap value if needed.)

The linking button worked perfectly fine in WordPress 6.2.

### Step-by-step reproduction instructions

1. In editor add a Columns block to post/page content.
2. Try to tweak "Block spacing" option to see you need to set up separate horizontal and vertical value even if you want to set the same value for both. This was working fine with WP6.2.

### Screenshots, screen recording, code snippet

![](https://github.com/WordPress/gutenberg/assets/2456735/12044446-08a3-4e47-8ae8-0f3bc7ca7d77)

### Environment info

- WP 6.3-RC2
- Gutenberg 16.3.0

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by hanneslsm** [comment]
> In my opinion setting up different horizontal and vertical block spacing/gap is more of an "advanced" approach, and in most cases setting up a single value for both is sufficient. 

May I know why you think that way? I use different horizontal and vertical settings very frequently (almost always), so I was very happy about the change. 
Fist, I change the vertical block spacing. Then, I look at the mobile version and can change (if needed - usually the default is fine) the vertical spacing. Changing the vertical and horizontal spacing automatically together and adding a grouping button would only add friction.

**Comment by webmandesign** [comment]
I've noticed that different horizontal and vertical gap settings are usually helpful for responsive design, just like you say. And that's perfectly fine. When I design my theme patterns and templates I use these different orientation settings all the time too. But I have no problem switching the un-link button in those cases (as a theme & pattern designer).

But I've noticed users usually simply set the block spacing without bothering setting up the different value for horizontal and vertical gap. Maybe because, they might not understand the difference: they might not understand that vertical gap only affects responsive, mobile view of Columns block, for example.

I also don't really get what would be the friction here. On WordPress 6.2 it was working the same way I propose (see the screenshot). I just would like to keep the already known consistency of the gap control. Especially when other dimension controls still allows similar functionality of grouping controls.

Please note I'm not asking for removal of separate horizontal and vertical gap setup. Just an option for grouping them together.

---

If this is not going to be in the WP core, I would at least appreciate some way to enabling this in the theme, maybe via `blocks.registerBlockType` filter tweaking block JSON `supports.spacing.blockGap`?

**Comment by annezazu** [comment]
@WordPress/gutenberg-design for feedback! From what I can remember, this was originally linked so I'm curious what changed and why here.

**Comment by jameskoster** [comment]
Yeah that seems like a bug to me.

**Comment by annezazu** [comment]
Thanks, Jay. Adding to the 6.3 board to see if we can address.

**Comment by ghorivipul97** [comment]
@jameskoster  @annezazu can you assign me, I will start to work from tomorrow.

**Comment by annezazu** [comment]
You can jump in whenever without needing to be assigned! Since this is in the 6.3 board and we're deeper in the cycle, let's leave unassigned for now in case someone can get to it ASAP.

**Comment by t-hamano** [comment]
Based on #49264 and #50660, it looks like the linked view has been removed after much discussion and consensus. So how about treating this issue as a feature improvement and moving it to "Punted to 6.4" ?

**Comment by jameskoster** [comment]
Removing the linked view for margin/padding seems fine – you often want different horizontal / vertical values. I'm not sure about spacing – my assumption is the most common use case would be equal values?

**Comment by jasmussen** [comment]
I think there's value in having the link/unlink button there just so things line up. And the axial block gap remains useful for things like navigation where I'd often have a larger horizontal gap than vertical gap for menus that wrap.

**Comment by ndiego** [comment]
> So how about treating this issue as a feature improvement and moving it to "Punted to 6.4" ?

I agree, and since 6.3 is coming out tomorrow, I am punting this to 6.4. For those following along, if changes are made, they will be available in the Gutenberg plugin well before 6.4.

**Comment by youknowriad** [comment]
Closing this old issue. While the behavior didn't change, it doesn't seem like it's bothering a lot of folks. We can reopen if there's more requests here.
